src/content/events/2023/symposium_wgc_dec23.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Nextflow Symposium 2023
+subtitle: A hybrid symposium held on Wellcome Genome Campus in Cambridge and online
+type: symposium
```

Comment:
sorry, the allowed types are: `"bytesize" | "talk" | "hackathon" | "training"`

Reply:
Updated to "talk"